backend/src/main/java/com/onwelo/practice/bts/entity/BankAccount.java

```diff
@@ -1,15 +1,18 @@
 package com.onwelo.practice.bts.entity;
 
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+
 import javax.persistence.*;
-import java.util.Collection;
+import java.util.List;
 
 @Entity
-@Table(name = "bank_account")
 public class BankAccount {
```

Comment:
Pomyślałbym czy nie robić dla tej encji doft-delete. Czyli zamiast usuwać encję chcielibyśmy jedynie ustawiać w niej flagę, która powie nam że encja jest nieaktywna/archiwalna. Ponownie, konta to raczej wrażliwe dane i nie chcemy się całkiem pozbywać ich historii.